### ISSUE TITLE
Paloma LightNode Client Proof Of Purchase on EVM

### ISSUE THREAD

**Issue by taariq** [VolumeFi/paloma#1865]
As a user, I want the Paloma LightClient download to prompt me for my ethereum purchase address to authenticate and verify my light node purchase so that I am able to prove just with my Ethereum address that I made the purchae.

**Acceptance Requirements**
1. Cosmwasm contract needs to emit EVM purchase details for the node sale
2. We can use geth to have user sighn to verify they own the address of the sale
3. We generate the Paloma address for user once authentication is complete and we prompt to store mnemonic
4. We store in Paloma the claim transaction for GRAINs

**Comment by verabehr** [comment]
split out pigeon part into a new ticket
